README.md

```diff
@@ -93,8 +94,7 @@ and time format variants you will need to handle.
 - If there are invalid UTF-8 characters in the `Notes` field, please
   replace them with the Unicode Replacement Character.
 
-You can assume that the input document is in UTF-8 and that any times
-that are missing timezone information are in US/Pacific. If a
+You can assume that the input document is in UTF-8. If a
```

Comment:
@sojeri How's this?

Reply:
It's still repeating information from above. 2 of 3 bullets above are about UTF-8, so do we still need to explicitly say assume the input is UTF-8? I would delete the first sentence and make the rest a single bullet